src/azure-cli/setup.py

```diff
@@ -141,9 +141,9 @@
     'fabric~=2.4',
     'javaproperties~=0.5.1',
     'jsondiff~=2.0.0',
-    'mycli~=1.22.2',
     'packaging>=20.9,<22.0',
     'PyGithub~=1.38',
+    'PyMySQL==1.0.2',
```

Comment:
```suggestion
    'PyMySQL~=1.0.2',
```
We usually use compatible release in `setup.py`